x-pack/test/functional/es_archives/security/flstest/data/mappings.json

```diff
@@ -7,7 +7,8 @@
         "runtime_customer_ssn": {
           "type": "keyword",
           "script": {
-            "source": "emit(doc['customer_ssn'].value + ' calculated at runtime')"
+            "lang": "painless",
+            "source": "if (doc['customer_ssn'].size() !== 0) { return emit(doc['customer_ssn'].value + ' calculated at runtime') } else { emit('') }"
```

Comment:
ES will throw an exception if there is no check if the 'customer_ssn' field exists.